src/doc/dev_manual/pr_create.rst

```diff
@@ -0,0 +1,152 @@
+
+If "Other" is checked, please describe the type of change in the space below.
+
+Testing
+~~~~~~~
+
+Replace the content of this section with a description of how the change was tested.
+
+
+.. _checklist:
+
+The Checklist
+~~~~~~~~~~~~~
+
+The Checklist serves as a list of suggested tasks to be performed before
+submitting the PR. Those that have been completed should be chedked off.
+Any items that do not relate to the PR should be deleted. For example, if
+the PR is not for a bugfix or feature, adding a test may not be required
+and this checklist item *should* be deleted.
+
+Notes
+~~~~~
+
+The template ends with two notes. The first is a note to squash merge the PR when it is ready to be merged. This allows you to turn several commits for one branch into one descriptive commit that encapsulates the work done on the branch. This keeps our git history clean and short. The second is a note to wait for the CI testing to pass before merging the PR. This is crucial because *if a CI test fails, then the branch should not be merged.* CI tests take approximately 30 minutes to complete.
+
+
+.. choose-a-reviewer:
+
+Reviewers
+---------
+
+GitHub will not allow non-owners to merge PRs into develop without a reviewer's
+approval. Non-owners will need at least one reviewer. Owner may merge a PR into
+develop without review. But, that doesn't necessarily mean they should.
+Follow the guidelines below to determine the need for and number of reviewers.
+Note, these guidelines serve as a "lower bound"; you may always add more
+reviewers to your PR if you feel that is necessary.
+
+
+No Reviewers (owners only)
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+If your changes are localized, you have satsified all the testing
+requirements and you are confident in the correctness of your changes
+(where correctness is measured by both the correctness of your code for
+accomplishing the desired task and the correctness of *how* you implemented
+the code according to VisIt_'s standard practices) then you may merge the PR
+without a reviewer *after* the CI tests pass.
+
+
+One reviewer
+~~~~~~~~~~~~
+
+If the changes have a broader impact or involve an unfamiliar area of VisIt_
+or existing behavior is being changed, then a reviewer should be added.
+
+Non-owners must always have at least one reviewer even if you satisfy all other
+guidelines for the *No Reviewers* case are satisfied.
+
+
+Two or more reviewers
+~~~~~~~~~~~~~~~~~~~~~
+
+If your changes substantially modify existing behavior or you are updating
+significant amounts of the code or you are designing new architectures or
+interfacess, then you should have at least two reviewers.
+
+
+Choosing Reviewers
+~~~~~~~~~~~~~~~~~~
+
+GitHub automatically suggests reviewers based on the blame data for the files
+you have modified. So unless you have a specific need for a specific reviewer,
+then you're probably safe to go with GitHub's suggestions.
+
+.. developer-process:
+
+Iteration Process
```

Comment:
Does it make sense to add a few remarks regarding *draft* PRs?

Reply:
I'm not sure how useful they are. I made a draft PR for the Remap operator, but it ended up working just like a normal PR....

Reply:
Its fine not to include in the docs...the impression I have is that leaving the PR in the draft state allows you to continue tweeking it without pestering reviewers for re-reviews due to changes that have been made. Reviewers may, at their option, review things there but nothing obligates them to do that until the PR is taken out of *draft* mode. When, that happens, reviewers are pestered. Again, fine if you don't wanna include draft PRs here. I think it makes sense NOT to include them too.